### PR TITLE
Update docs

### PR DESCRIPTION
**Pull request by RemyLau**

### CODE REVIEW

File: docs/source/notes/custom.rst [hunk 1]
```diff
@@ -34,19 +32,19 @@ set up the working files by
...
+   custom.edgelist_to_nodeorder("path/to/net.edg", "path/to/data",
                            "your_net_name")
 
-   # Set up adjacency feature
+   # Create adjacency feature representation
    custom.edgelist_to_matrix("path/to/net.edg", "path/to/data",
                              "your_net_name", "Adjacency")
 
-   # Alternatively, set up influence feature
+   # Create influence feature representation
```
Comment (RemyLau) [inline docs/source/notes/custom.rst:42]: Should we keep ``Alternatively`` since the user should choose between the two?

**Comment by RemyLau** [comment]
### Initial review

- [ ] Flake8 errors to be fixed
  ```txt
  geneplexus/geneplexus.py:333:1: D205 1 blank line required between summary line and description
  [16](https://github.com/krishnanlab/PyGenePlexus/runs/6868217028?check_suite_focus=true#step:5:17)
    geneplexus/geneplexus.py:333:1: D400 First line should end with a period
  [17](https://github.com/krishnanlab/PyGenePlexus/runs/6868217028?check_suite_focus=true#step:5:18)
    geneplexus/__init__.py:1:1: D205 1 blank line required between summary line and description
  [18](https://github.com/krishnanlab/PyGenePlexus/runs/6868217028?check_suite_focus=true#step:5:19)
    geneplexus/__init__.py:1:1: D400 First line should end with a period
  [19](https://github.com/krishnanlab/PyGenePlexus/runs/6868217028?check_suite_focus=true#step:5:20)
    geneplexus/__init__.py:37:121: E501 line too long (126 > 120 characters)
  [20](https://github.com/krishnanlab/PyGenePlexus/runs/6868217028?check_suite_focus=true#step:5:21)
    geneplexus/custom.py:17:1: D205 1 blank line required between summary line and description
  [21](https://github.com/krishnanlab/PyGenePlexus/runs/6868217028?check_suite_focus=true#step:5:22)
    geneplexus/custom.py:17:1: D400 First line should end with a period
  ```
- [ ] Need to update license tag in ``setup.cfg`` accordingly
- [ ] Move figures to ``docs/figures`` for better organization if possible
- [ ] Fix linewidth
  - [ ] ``docs/source/notes/api.rst``
  - [ ] ``docs/source/notes/cli.rst``
  - [ ] etc

File: docs/source/notes/custom.rst (hunk 1 above)
Comment (RemyLau) [reply docs/source/notes/custom.rst:42]: Actually, nevermind, this is fine - both can be generated but the user only needs to choose when executing geneplexus.

**Comment by ChristopherMancuso** [comment]
- [ ] Fix equations in representations.rst
- [ ] Fix equations in similarities.rst

**Pull request merged**